.github/workflows/build.yml

```
@@ -0,0 +1,114 @@
+          git checkout -B release-$TAG_VERSION
+
+          # Clear everything except .git
+          find . -maxdepth 1 ! -name '.git' ! -name '.' -exec rm -rf {} +
+
+          # Copy addon contents to root for submodule use
+          git checkout "$COMMIT_SHA" -- README.md LICENSE addons/gecs
+          cp README.md addons/gecs/
+          cp LICENSE addons/gecs/
+          cp -r addons/gecs/* .
+          rm -rf addons
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)